ios/Handlers/RegistrationHandler.swift

```diff
@@ -19,19 +19,7 @@ class RegistrationHandler: NSObject, BrowserHandlerToRegisterHandlerProtocol {
     var browserConntroller: BrowserHandlerProtocol?
     var signUpCompletion: ((Bool, ONGUserProfile?, Error?) -> Void)?
     let createPinEventEmitter = CreatePinEventEmitter()
-
-    func presentBrowserUserRegistrationView(registrationUserURL: URL) {
-        if(browserConntroller != nil) {
-            browserConntroller?.handleUrl(url: registrationUserURL)
-        } else {
-            if #available(iOS 12.0, *) {
-                browserConntroller = BrowserViewController(registerHandlerProtocol: self)
-                browserConntroller?.handleUrl(url: registrationUserURL)
-            } else {
-              // Fallback on earlier versions
-            }
-        }
-    }
+    let registrationEventEmitter = RegistrationEventEmitter()
```

Comment:
if not needed outside please make it private <BR>
`private let registrationEventEmitter = RegistrationEventEmitter()`